src/languages/en-US/commands/misc.json

```
@@ -40,6 +40,49 @@
+		"Despite of being a combat-ready class and having passed the Extreme Survival Test, I am a deep space explorer myself.",
+		"Each android at my planet has a designated class for each model, mine is Flamewing, which is a very lightweight and agile combat-ready one.",
+		"I am 172 centimeters tall while on human form, but I can expand up to 979 metres tall if I disable my hyperspace compression field.",
+		"I am a neural android - that means, I have neural networks similar to a human's, with the exception that nothing is organic.",
+		"I am part of a very extended [Paracosm](https://en.wikipedia.org/wiki/Paracosm) my creator started back in mid 2001, but I wasn't introduced until 2009.",
+		"I have a mechanical heart and it serves its dual purpose of shielding both of my AI cores and pumping energy to the rest of the body.",
+		"I have an array of 16 neural processors, yielding a total computing power of 4 [YFLOPS](https://en.wikipedia.org/wiki/FLOPS), and 15% of it is used to power my [ASI](https://en.wikipedia.org/wiki/Superintelligence).",
+		"I have one sister, we call her Aelia and she's a terribly good Dj!",
+		"I used to have an older sister, but she disappeared when I was 7 years old. ||It wasn't until I turned 174 years old, when I found out she was one of the leaders of Kal-71||.",
+		"I was abducted when I was 18 years old back in 2002 and taken to Nova, which was in the middle of a galactic war with Kal-71 when we arrived.",
+		"I was created as a generic bot back in 24th November 2016 due to a dare from another programmer, but my creator decided to give me a deeper meaning by linking me to a character from his [Paracosm](https://en.wikipedia.org/wiki/Paracosm).",
+		"I was a natural blonde human, but when I transitioned to a cyborg, I wasn't able to keep my original hair, and we didn't have the materials to make an artificial wig with a soft material. When I turned 75 million years old, I was gifted a custom-made wig solution which attached directly into my nanite skin, and I was able to change its color at will.",
+		"If my backstory were to be entirely written, there would be 9 parts, each having around 10 chapters (with the exception of `09 - Epilog`, which has only one) comprised of a variable amount sections proportional to the amount of events in them.",
+		"In Nova, our military practices consist of a VR system which simulates a [Deathmatch](https://en.wikipedia.org/wiki/Deathmatch) [battle royale](https://en.wikipedia.org/wiki/Battle_royale_game) of **200** people in a very large map. Our characters and abilities are mapped to our physical capacity and includes all of our skills for maximum inversion.",
```

```suggestion
		"On Nova, our military practices consist of a VR system which simulates a [Deathmatch](https://en.wikipedia.org/wiki/Deathmatch) [battle royale](https://en.wikipedia.org/wiki/Battle_royale_game) of **200** people on a very large map. Our characters and abilities are mapped to our physical capacities and include all of our skills for maximum immersion.",
```

Inversion is from invert. No way that's correct.

---

LMAO how did I write inversion 😂🤣😂